view.go

```
@@ -63,8 +76,8 @@ func renderStyles() {
 
 	r, err := updateTermRendered(resultStyle)
 	if err != nil {
-		// hoping this doesn't happen as most commands here suceed without issue
-		panic(err)
+		fmt.Fprintf(os.Stderr, "Failed to update term renderer in renderStyles: %v\n", err)
+		return
```

Maybe return an error here too.